### PR TITLE
fix: remove irrelevant pages from legacy docs for interim period

### PR DESCRIPTION
**Pull request by nlc616**
## Why?

Adds some very simple tweaks (removes certain pages for the time being since we are getting some new traffic from the Base hackathon developers, who don't need to know about the .co migration process) and adds an updated headline on the overview page. 

Harrison has requested we make this slightly easier for those Base hackathon developers to dive into. 

## How?

- Done A (replace with a breakdown of the steps)
- Done B
- Done C

## Tickets?

- [Ticket 1](the-ticket-url-here)
- [Ticket 2](the-ticket-url-here)
- [Ticket 3](the-ticket-url-here)

## Contribution checklist?

- [ ] The commit messages are detailed
- [ ] The `build` command runs locally
- [ ] Assets or static content is linked and stored in the project
- [ ] Document filename is named after the slug
- [ ] You've reviewed spelling using a grammar checker
- [ ] For guides or references, you've tested the commands and steps
- [ ] You've done enough research before writing

## Security checklist?

- [ ] Injection has been prevented (parameterized queries, no eval or system calls)
- [ ] The UI is escaping output (to prevent XSS)
- [ ] Sensitive data has been identified and is being protected properly

## References?

Optionally, provide references such as links

## Preview?

Optionally, provide the preview url here

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -17,11 +17,9 @@ tags:
...
 
-Welcome to the documentation for the alpha of [Fleek.xyz](https://fleek.xyz). Whether you are an expert or an absolute beginner, you'll find your answers here.
-
-Fleek is a Web3 infrastructure platform that surfaces protocols like Fleek Network, IPFS, Arweave, Filecoin, POKT, ENS, and more. Use them independently or combine them together to create awesome new experiences.
+Fleek is an edge-optimized cloud platform where users can prepare, host and deploy Web sites and applications. It enables developers to build fast, performant apps.
 
 - Need support? Visit our [Support Center and Knowledge Hub.](https://support.fleek.xyz/hc/en-us)
 - Feedback? Enter our [Discord channel](https://discord.gg/fleek) to have a direct line with our developers.
-- Would like to find a bit more of what we do? In our [blog](https://blog.fleek.xyz) you can find more information.
+- Want more info on what Fleek does? You can find more information on our [blog](https://blog.fleek.xyz).
```
Comment (punkbit) [inline docs/index.md:24]:
```suggestion
- Want more info on what Fleek does? You can find more information on our [blog](https://fleek.xyz/blog).
```

**Review by punkbit — CHANGES_REQUESTED** [review]
- The `docs/Accounts/pat.md` is linked from https://app.fleek.xyz/profile/settings/personal-access-tokens 
- Update title in accordance to the contribution guideline

File: docs/index.md (hunk 1 above)
Comment (nlc616) [reply docs/index.md:24]: Not yet - this one will still be the pre-Tuesday site so the blog will still exist on `blog.` while it's live.

**Comment by nlc616** [comment]
> * The `docs/Accounts/pat.md` is linked from https://app.fleek.xyz/profile/settings/personal-access-tokens
> * Update title in accordance to the contribution guideline

Updated title. Broken link is OK for the 2-3 days that this is transitioning over; I'll make sure we have a new version of this up in the docs releasing early next week. 

EDIT: Note: I checked the other titles just now and can confirm this approach is now consistent with the others, but our PR title verification check and convention-pr-title check in this repo are failing on a missing node module. It doesn't matter since this repository is not going to be maintained once the new docs go live.

**Review by punkbit — APPROVED** [review]

**Pull request merged**